test/e2e/theia_clickhouse_test.go

```diff
@@ -0,0 +1,358 @@
+// limitations under the License.
+
+package e2e
+
+import (
+	"crypto/rand"
+	"database/sql"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"antrea.io/theia/pkg/theia/commands"
+)
+
+const (
+	getDiskInfoCmd   = "./theia clickhouse status --diskInfo"
+	getTableInfoCmd  = "./theia clickhouse status --tableInfo"
+	getInsertRateCmd = "./theia clickhouse status --insertRate"
+	insertQuery      = `INSERT INTO flows (
+                   flowStartSeconds,
+                   flowEndSeconds,
+                   flowEndSecondsFromSourceNode,
+                   flowEndSecondsFromDestinationNode,
+                   flowEndReason,
+                   sourceIP,
+                   destinationIP,
+                   sourceTransportPort,
+                   destinationTransportPort,
+                   protocolIdentifier,
+                   packetTotalCount,
+                   octetTotalCount,
+                   packetDeltaCount,
+                   octetDeltaCount,
+                   reversePacketTotalCount,
+                   reverseOctetTotalCount,
+                   reversePacketDeltaCount,
+                   reverseOctetDeltaCount,
+                   sourcePodName,
+                   sourcePodNamespace,
+                   sourceNodeName,
+                   destinationPodName,
+                   destinationPodNamespace,
+                   destinationNodeName,
+                   destinationClusterIP,
+                   destinationServicePort,
+                   destinationServicePortName,
+                   ingressNetworkPolicyName,
+                   ingressNetworkPolicyNamespace,
+                   ingressNetworkPolicyRuleName,
+                   ingressNetworkPolicyRuleAction,
+                   ingressNetworkPolicyType,
+                   egressNetworkPolicyName,
+                   egressNetworkPolicyNamespace,
+                   egressNetworkPolicyRuleName,
+                   egressNetworkPolicyRuleAction,
+                   egressNetworkPolicyType,
+                   tcpState,
+                   flowType,
+                   sourcePodLabels,
+                   destinationPodLabels,
+                   throughput,
+                   reverseThroughput,
+                   throughputFromSourceNode,
+                   throughputFromDestinationNode,
+                   reverseThroughputFromSourceNode,
+                   reverseThroughputFromDestinationNode)
+                   VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?, ?, ?, ?,
+                           ?, ?, ?, ?, ?, ?, ?)`
+	recordPerCommit = 1000
+	insertInterval  = 1
+	threshold       = 25
+	MaxInt32        = 1<<31 - 1
+)
+
+var targetTable = map[string]string{
```

Comment:
I'm a little bit confused about this variable name. Is this a map of column numbers and table names? Better to consider renaming it.

Reply:
Yes. I've renamed it. Thanks.